packages/swapper/src/swappers/JupiterSwapper/errors.ts

```diff
@@ -0,0 +1,23 @@
+import type { SwapperErrorMapping } from '../baseErrors'
+import { SWAPPER_ERRORS } from '../baseErrors'
+
+export const JUPITER_ERROR_VALUES = {
+  SLIPPAGE_EXCEEDED: 'SlippageToleranceExceeded',
+  UNDER_MINIMUM_AMOUNT: 'RequireGteViolated',
+  CONSUMED_MORE_FEES: 'exceeded CUs meter',
+} as const
+
+export const JUPITER_ERRORS: SwapperErrorMapping[] = [
```

Comment:
preferably-blocking: This is reundant.
1. this is a K,V mapping which 1/1 maps the keys and values of `JUPITER_ERROR_VALUES` 
2. We're effectively reinventing the wheel and creating a poor man's flavour of `Object`

Reply:
Changed in [17f168a](https://github.com/shapeshift/web/pull/8433/commits/17f168a1ba52efd3e9b3f5e55e1b8f1e2266db37)